src/diffpy/labpdfproc/tools.py

```diff
@@ -34,3 +34,36 @@ def set_wavelength(args):
         return WAVELENGTHS[args.anode_type]
     else:
         return WAVELENGTHS["Mo"]
+
+
+def _load_key_value_pair(s):
+    items = s.split("=")
+    key = items[0].strip()
+    if len(items) > 1:
+        value = "=".join(items[1:])
```

Comment:
this is making some assumptions about which is the "splitting" = sign.  This is ok, but let's let the people know what hte rule is that we are applying (equals sign not allowed in key name?) in the help message to the argparse.  I htink adding a few examples to the help might be good too, but don't do toast for breakfast but something topical, like `-u "facility=NSLS II" beamline=28ID-2`